main/thumbar.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  disable,
+  enable,
+  onPlayerPause,
+  onPlayerPlay
+}
+
+/**
+ * On Windows, add a "thumbnail toolbar" with a play/pause button in the taskbar.
+ * This provides users a way to access play/pause functionality without restoring
+ * or activating the window.
+ */
+
+var path = require('path')
+var config = require('../config')
+
+var windows = require('./windows')
+
+/**
+ * Show the Windows thumbnail toolbar buttons.
+ */
+function enable () {
+  update(false)
+}
+
+/**
+ * Hide the Windows thumbnail toolbar buttons.
+ */
+function disable () {
```

Comment:
I think the function's name is misleading, since it's supposed to be called only on player close. If we call that function in some other context and then pause or resume playback, the thumbar will reappear.